README.md

```diff
@@ -88,6 +88,6 @@ Enable the module. That's it.
 Permissions
 -----------
 
-Someone able to access the cube will see statistics for **all** available
```

Comment:
Please make sure to align text the same way as it's the case for the remaining file. Also, while you're at it please fix `at lease` :smile: